test/e2e/save_test.go

```diff
@@ -111,6 +111,24 @@ var _ = Describe("Podman save", func() {
 		Expect(save.ExitCode()).To(Equal(0))
 	})
 
+	It("podman save to directory with --compress but not use docker-dir and oci-dir", func() {
+		if rootless.IsRootless() && podmanTest.RemoteTest {
+			Skip("Requires a fix in containers image for chown/lchown")
+		}
```

Comment:
You can use `SkipIfRootless("...")` instead.

Reply:
Can you elaborate on what needs to be fixed?  If there is no open issue for that yet, could you open one and reference it as `#issueNumber` in the skip message?

Reply:
> You can use `SkipIfRootless("...")` instead.

This condition is copy from this file, I think is right

Reply:
> Can you elaborate on what needs to be fixed? If there is no open issue for that yet, could you open one and reference it as `#issueNumber` in the skip message?
Fixes https://github.com/containers/podman/issues/10024

Reply:
@zhangguanzhang yes, your test is correct, but if you could convert to the SkipIfRootless as @vrothberg showed, it would be appreciated.  That way if we ever need to quickly find our rootless tests, we have one convenient tag to do so.

Reply:
@TomSweeneyRedHat  These two conditions are parallel logical AND, I think SkipIfRootless should not be used